x/cvm/client/cli/query.go

```diff
@@ -311,3 +314,39 @@ func GetCmdAddressTranslate(queryRoute string, cdc *codec.Codec) *cobra.Command
 		},
 	}
 }
+
+// GetAccountCmd returns a query account that will display the state of the
+// account at a given address.
+func GetAccountCmd(cdc *codec.Codec) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "contract [address]",
+		Short: "Query contract info",
+		Long:  "Query contract info by address, revert to query normal account info if the address is not a contract",
+		Args:  cobra.ExactArgs(1),
+		RunE: func(cmd *cobra.Command, args []string) error {
+			cliCtx := context.NewCLIContext().WithCodec(cdc)
+			address := args[0]
+
+			var account exported.Account
+			res, _, err := cliCtx.QueryWithData(fmt.Sprintf("custom/cvm/account/%s", address), nil)
+			if err != nil {
+				return err
+			}
+			cdc.MustUnmarshalJSON(res, &account)
+
+			baseAcc, ok := account.(*auth_types.BaseAccount)
```

Comment:
This assertion should be removed because `account` might not be a `BaseAccount`

Reply:
I think the assertion should be moved into QueryCVMAccount because CVMAccount inherits from BaseAccount.
Here I can keep the exported Account because it might be a VestingAccount, right?

Reply:
I think so, yeah. Thank you.